CMakeLists.txt

```diff
@@ -36,14 +36,8 @@ option(COSE_C_INCLUDE_ENCRYPT0 "Include COSE_ENCRYPT0" ON)
 option(COSE_C_INCLUDE_MAC "Include COSE_MAC" ON)
 option(COSE_C_INCLUDE_MAC0 "Include COSE_MAC0" ON)
 option(COSE_C_INCLUDE_SIGN "Include COSE_SIGN" ON)
-option(COSE_C_INCLUDE_SIGN0 "Include COSE_SIGN0" ON)
 option(COSE_C_INCLUDE_SIGN1 "Include COSE_SIGN1" ON)
 
-if(NOT COSE_C_INCLUDE_SIGN0)
-  message("Use COSE_C_INCLUDE_SIGN1 rather than COSE_C_INCLUDE_SIGN0")
-  set(COSE_C_INCLUDE_SIGN1 ${COSE_C_INCLUDE_SIGN0})
-endif()
-
 # Set the output of the libraries and executables.
 set(EXECUTABLE_OUTPUT_PATH ${PROJECT_BINARY_DIR}/bin)
 set(LIBRARY_OUTPUT_PATH ${PROJECT_BINARY_DIR}/lib)
```

Comment:
@jimsch  is definition below `add_definitions(-DUSE_ARRAY)` still needed?

Reply:
No that is long since history.